.circleci/config.yml

```diff
@@ -416,6 +428,15 @@ jobs:
           root: ~/project
           paths:
             - shared-coverage/index_detail
+  remaining_tests:
+    <<: *config
+    steps:
+      - *attach_workspace
+      - run: ./$BOOST_DIR/libs/geometry/.circleci/run_test.sh
```

Comment:
What was the intention here? This script requires parameters, without it it will not run anything.